runtime/fundamentals/modules.md

```diff
@@ -0,0 +1,450 @@
+---
+title: "Using modules in Deno"
+oldUrl: 
+  - /runtime/manual/basics/modules/
+  - /runtime/manual/basics/modules/integrity_checking/
+  - /runtime/manual/basics/modules/module_metadata/
+  - /runtime/manual/basics/modules/proxies/
+  - /runtime/manual/basics/modules/publishing_modules/
+  - /runtime/manual/basics/modules/reloading_modules/
+  - /runtime/manual/advanced/publishing/dnt/
+  - /runtime/manual/advanced/publishing/
+  - /runtime/manual/basics/vendoring/
+  - /runtime/manual/node/cdns.md
+  - /runtime/manual/advanced/http_imports/
+---
+
+Deno standardizes the way modules are imported in both JavaScript and TypeScript
+using the
+[ECMAScript module standard](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules)
+by default. In line with this standard, **file names must be specified in
```

Comment:
"Deno standardizes the way modules are imported..." is odd phrasing

How about "Deno uses ECMAScript modules exclusively. That is, `import` rather than `require`."

Maybe add a bit of context about why -- something like "since 2015(?) ES Modules have been part of the specification that defines JavaScript. As such ES Modules work in the browser, CommonJS does not. Deno's goal is to narrow the gap between browser and server."